.github/actions/templates/validateModulePester/action.yml

```
@@ -1,5 +1,5 @@
-name: 'Execute general module tests'
-description: 'Execute general module tests (if any)'
+name: 'Execute global Pester module tests'
+description: 'Execute global Pester module tests (if any)'
```

How are we referencig to the first and second group of tests? Should it be "global test" for the former and "API versions' tests" for the latter? In this case I'd suggest
```suggestion
name: 'Execute Pester module tests'
description: 'Execute Pester module tests (if any)'
```